uniflow/op/model/layout_utils.py

```diff
@@ -0,0 +1,348 @@
+"""Layout Module."""
+from typing import List
+
+import cv2
+import numpy as np
+import onnxruntime
+from huggingface_hub import hf_hub_download
+
+
+def preprocess(img, input_size, swap=(2, 0, 1)):
```

Comment:
nit: please add all missing typehint and docstring (argument, return, ...) for this Non Maximum Suppression part.